agent/bench-scripts/pbench-linpack

```diff
@@ -113,24 +117,42 @@ while true; do
 done
 verify_common_bench_script_options ${tool_group} ${sysinfo}
 
-ver="$(pbench-config version ${benchmark})"
-if [[ -z "${ver}" ]]; then
-	error_log "${script_name}: package version is missing in config file"
-	exit 1
-fi
-if [[ -z "${linpack_dir}" ]]; then
-	linpack_dir="/usr/local/${script_name}-${ver}/benchmarks/linpack"
-	linpack_dir_kind="default"
-else
-	linpack_dir_kind="provided"
+function pre_check {
+	${pbench_bin}/bench-scripts/driver/linpack --pre-check ${ver}
+	return ${?}
+}
```

Comment:
Where is `${ver}` defined?  Better yet, should it be a positional argument (i.e., `${1}`) to this function?

Should we be checking the argument (like, making sure there is one...) before passing it on to the driver?

(FWIW, I don't think that the `return ${?}` actually accomplishes anything -- I think that we get the same result if you omit the argument or the entire statement.)

Reply:
Other than the nit about the `return` statement, you've largely addressed these.